api/v1alpha1/iperf3_types.go

```diff
@@ -20,21 +20,55 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// Iperf3Spec defines the Iperf3 Benchmark Job
+// Iperf3ConfigurationSpec contains configuration parameters
+// with scheduling options for the both the iperf3 client
+// and server instances.
+type Iperf3ConfigurationSpec struct {
+	// Command line arguments appended to the
```

Comment:
Should start with "CmdLineArgs"

Reply:
I also read the suggestion in vs-code quick fix, but it is not a strict requirement.
One can find examples of not repeating the variable name in the kubernetes lib too:
https://godoc.org/k8s.io/apimachinery/pkg/apis/meta/v1#DeleteOptions

Reply:
https://blog.golang.org/godoc-documenting-go-code

> Notice this comment is a complete sentence that begins with the name of the element it describes. This important convention allows us to generate documentation in a variety of formats, from plain text to HTML to UNIX man pages, and makes it read better when tools truncate it for brevity, such as when they extract the first line or sentence.